.github/workflows/maintenance.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+  schedule:
+    - cron: '*/15 * * * *'
```

Comment:
This seems aggressive for how frequently these things change.

Reply:
It won't change anything if nothing has changed, and it'll just update the same PR if anything has changed, so I considered running frequently and we could just merge it right before releases. Every 15 minutes seems to still be getting `ThrottlingException when calling the GetProducts operation` so trying every 20 minutes now

Reply:
We might be able to use [pricing files](https://github.com/aws-cloudformation/cfn-python-lint/pull/1535) instead of the pricing API?

Reply:
[can also temporarily disable Github actions](https://github.com/aws-cloudformation/cfn-python-lint/actions?query=workflow%3A.github%2Fworkflows%2Fmaintenance.yaml) if we need to patch [Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) bugs any week

Reply:
@miparnisari / anyone else receiving `Run failed: .github/workflows/maintenance.yaml` emails in their fork because they don't have `secrets.AWS_ACCESS_KEY_ID`/`secrets.AWS_SECRET_ACCESS_KEY` set: [GitHub Actions notifications can be disabled](https://github.com/settings/notifications) (doesn't look more configurable yet), individual workflows can be disabled, or [encrypted](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets) AWS credentials set